src/ALaDyn.F90

```diff
@@ -54,7 +54,7 @@ program ALaDyn
  tnow=tstart
  ! iter=last_iter
  iter=0
- call diag_part_dist
+ !call diag_part_dist
```

Comment:
Maybe this removal is a typo?

Reply:
Yes, it's a typo due to a quick merging of two files I didn't check.